src/format/parsed.rs

```diff
@@ -575,6 +577,175 @@ impl Parsed {
         set_if_consistent(&mut self.offset, i32::try_from(value).map_err(|_| OUT_OF_RANGE)?)
     }
 
+    /// Get the 'year' field if set.
+    ///
+    /// See also [`set_year`](Parsed::set_year).
+    #[inline]
+    pub fn year(&self) -> Option<i32> {
```

Comment:
If we're going to have these getters we can also refer to these in those earlier docs where you didn't want to refer to the fields directly?

Reply:
I'll add similar "See also ..." lines to the `set` methods.

Reply:
That's not really what I'm after. I don't like `'year divided by 100'" "descriptions" because they are kind of imprecise references to something that actually has a precise reference in the code, with more information. So IMO we should get rid of the "descriptions" in favor of precise references.

Reply:
I am not sure I like linking to a method as if it is a field any better.
Added as a commit. Is this more what you had in mind?

Reply:
Can you write one line as an example of a sentence? I only didn't do it until now because I can't come up with something that makes sense to me.

Reply:
Example of our current options:

Set the 'year' field to the given value.
Set the `year` field to the given value.
Set the `year()` field to the given value.

Set the 'year divided by 100' field that is part of an ISO 8601 week date to the given value.
Set the `isoyear_div_100` field, that is part of an ISO 8601 week date, to the given value.
Set the `isoyear_div_100()` field, that is part of an ISO 8601 week date, to the given value.

Set the 'week number starting with Sunday' field to the given value.
Set the `week_from_sun` week number field to the given value.
Set the `week_from_sun()` week number field to the given value.

Reply:
I think I'd prefer the second option in the end. We'll eventually get [read-only fields](https://rust-lang.github.io/rfcs/3323-restrictions.html) at which point we could conceivably make these `pub mut(crate)` (or however it ends up being spelled).